docs/ReferenceArrayInput.md

```diff
@@ -328,6 +330,76 @@ Then to display a selector for the post tags, you should call `<ReferenceArrayIn
 <ReferenceArrayInput source="tags_ids" reference="tags" />
 ```
 
+## `readOnly`
+
+If `true`, the default input is in read-only mode and the user can't change the value.
+
+{% raw %}
+
+```jsx
+<ReferenceArrayInput
+    source="tags_ids"
+    reference="tags"
+    readOnly
+/>
+```
+
+{% endraw %}
+
+**Warning**: The `readOnly` prop refer to the default [`<AutocompleteArrayInput>`](./AutocompleteArrayInput.md) child. If you defined `children` you have to specify the `readOnly` prop on each input:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
```suggestion
**Warning**: The `readOnly` prop refers to the default [`<AutocompleteArrayInput>`](./AutocompleteArrayInput.md) child. If you pass a child component, you'll have to specify the `readOnly` prop on each input:
```